### PR TITLE
Added swagger doc drf-yasg package

### PR DESCRIPTION
**Pull request by KunalAnkur**
### Description
By fixing this issue I have used drf-yasg package in which we can only interact via swagger user interface.

**Type of Change:** 
- Code

**Code/Quality Assurance Only** 
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
You can test just by running python manage.py runserver after installing all the requirements.
till now there is no API in the app so we can just see the UI of swagger doc at http://localhost:8000/swagger url.

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -6,3 +6,7 @@ Pillow==7.1.1
 python-decouple==3.3
 pytz==2019.3
 sqlparse==0.3.1
+
+
+drf-yasg
+
```
Comment (naveennvrgup) [inline requirements.txt:12]:
```suggestion
drf-yasg
```

These empty extra lines are not needed here :)

File: server/urls.py [hunk 2]
```diff
@@ -16,10 +16,34 @@
...
+)
+
+
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/query/',include('query.urls')),
     path('api/users/',include('users.urls')),
     path('api/org/',include('org.urls')),
     path('api/tasks/',include('tasks.urls')),
+    path('swagger', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui')
```
Comment (naveennvrgup) [inline server/urls.py]:
```suggestion
    path('', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui')
```

Lets keep this swagger docs in the home route

File: server/urls.py [hunk 3]
```diff
@@ -16,10 +16,34 @@
...
+      title="teesco-API",
+      default_version='v1',
+      description="Test description",
+      terms_of_service="https://www.google.com/policies/terms/",
+      contact=openapi.Contact(email="contact@snippets.local"),
+      license=openapi.License(name="BSD License"),
+   ),
+   public=True,
+   permission_classes=(permissions.AllowAny,),
+)
```
Comment (naveennvrgup) [inline server/urls.py:38]:
```suggestion
schema_view = get_schema_view(
   openapi.Info(
      title="teesco-API",
      default_version='v1',
      description="Test description",
      terms_of_service="https://https://ecell.nitrr.ac.in/terms",
      contact=openapi.Contact(email="ecellnitrr.opensource@gmail.com"),
      license=openapi.License(name="Apache License 2.0"),
   ),
   public=True,
   permission_classes=(permissions.AllowAny,),
)
```

**Review by naveennvrgup — COMMENTED** [review]
I have made a few suggestions please address those. Great job on your first PR! :fire: 

These are few areas where you can improve upon:
1. Follow commit guidelines
1. dont make PR from you dev branch! work on separate branch for each issue
1. Good to see the PR template. Please check the relevant options that applies to this PR

File: requirements.txt (hunk 1 above)
Comment (KunalAnkur) [reply requirements.txt:12]: yes sir

File: server/urls.py (hunk 2 above)
Comment (KunalAnkur) [reply server/urls.py]: okay sir

File: server/urls.py (hunk 3 above)
Comment (KunalAnkur) [reply server/urls.py:38]: okay sir i got it

**Comment by KunalAnkur** [comment]
Thank you so much for your suggestion. Next time I will keep this in mind

**Comment by naveennvrgup** [comment]
> I have made a few suggestions please address those. Great job on your first PR! 
> 
> These are few areas where you can improve upon:
> 
> 1. Follow commit guidelines
> 2. dont make PR from you dev branch! work on separate branch for each issue
> 3. Good to see the PR template. Please check the relevant options that applies to this PR

@KunalAnkur The comments are to be marked as resolved by the commentator(@me) not the replier. Please make commits and push the requested changes. Until then I will restrain from merging. :innocent: 

To be clear, the 3 areas of improvement applies for the next PR while the comments apply for the current PR.

**Comment by sumit-badsara** [comment]
As there is another PR regarding the same issue. So, closing this one!

**Pull request closed without merging**